### DISCUSSION TITLE
Stack Sizes for Item Slots

### DISCUSSION THREAD

**Discussion by kevinq2000** [peter-kish/gloot#186] [Q&A]
Thank you so much for the hard work in this addon - I'm finding it very helpful for getting my systems working. 

I've just noticed an issue, and I'm not sure if I'm doing something wrong. I'd like to have a stack of usable items in an Item Slot box - like a apples that can be eaten, and the stack gets smaller when you eat them.

If I add a stack of items to a CtrlItemSlot box, it won't show the stack size until I start removing individual items. You can see part of this in the examples that come with the addon - if you open the inventory_grid_stacked_ex_transfer.tscn file and run the current scene, dragging either the stack of armor or the books into the Item Slot fails to show the stack size. In my own implementation, I've setup a button to drop single copies of the "equipped" item into the world. If I have a stack larger than 2, once I start dropping, the number shows up in the item slot box. But I can't figure out how to get it to show up without changing the stack size. Thoughts?

Thanks!

**Comment by peter-kish** [comment]
This turned out to be a bug in the addon. Should be fixed now in 55a7dedc3875c10679300ad485d7d1a0935c5dea.
Thanks for letting me know!

**Comment by kevinq2000** [comment]
Thanks! It's a super-helpful addon.

K
